lib/Doctrine/DBAL/Logging/NullLogger.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Doctrine\DBAL\Logging;
+
+/**
+ * A SQL logger that does nothing.
+ */
+class NullLogger implements SQLLogger
```

Comment:
Could be final.

Reply:
OK!

Reply:
Should I make the other logger classes `final`, by the way?